app/models/events/pull_request.rb

```diff
@@ -46,6 +46,7 @@ class PullRequest < ApplicationRecord
                        inverse_of: :pull_request
     has_many :events, as: :handleable, dependent: :destroy
     has_one :merge_time, dependent: :destroy
+    has_one :pull_request_size, dependent: :destroy
```

Comment:
why isn't this an integer column?

Reply:
Well, that's a great question.
When I was building this, I noticed that the other two distribution metrics had their own separated model (`MergeTime` and `CompletedReviewTurnaround`). So I made the decision of creating one for this new metric just to stick with the pattern and make it more clear, even when it's less efficient (in terms of performance and storage).
I'm not too sure about that decision, though, and it's easy to try the other option, so I'd love to hear your thoughts!

Reply:
I see.. I like to idea of keeping things consistent. I guess there is a reason why we have models instead of just values 🤔 do you remember why was it @santiagovidal @hvilloria ?

Reply:
I don't quite remember if we considered it at that moment, but I guess they weren't as straightforward as this one.
Both review turnaround and merge time don't occur at the same time as the PR creation for obvious reasons, so we would need to store when the PR was merged or reviewed as well, or to consider the time the PR was opened for it.
Still, I can't think of a strong reason against doing it that way 🤔 
I think @hdamico came up with the idea in the first place, so maybe he could also share his input?

Reply:
Having MergeTime and ReviewTurnaround in different tables is because these two metrics are calculated between two records, so I don't think those have to go in a table column also, when it comes to query them, it is easier to get the data that we want. I don' t really remember if this is a pattern, I only remember doing this for convenience. Maybe @santiagovidal or @hvilloria remember something

Reply:
If you guys believe having ReviewTurnaround and MergeTime in a column inside PR table im good with that.

Reply:
But the MergeTime belongs to a PullRequest, so it could be a value in that table. And the ReviewTurnaround belongs to a ReviewRequest so the value could be just there.

That said, I'm ok keeping things this way for now. If at some point we see a benefit to do the change we can discuss it further